### PR TITLE
New View for Trial project Feature #44304

### PR DESCRIPTION
**Pull request by trishasalas**
As stated in Slack "currently, the select in the tribe bar doesn’t work nor does the paging nav"
Thanks!

### CODE REVIEW

**Comment by borkweb** [comment]
A few general notes to help point you toward getting the Ajax and other stuff working:

##### In `src/functions/template-tags/general.php`

* Add a case for this new view in `tribe_is_ajax_view_request()` (for Ajax stuff)
* Add a case for this new view in `tribe_events_the_header_attributes()` to ensure the right data attributes are in place.

##### In `src/functions/template-tags/link.php`

* Duplicate the `tribe_get_listview_past_link()` function for your new view

File: src/resources/css/tribe-events-full.css [hunk 1]
```diff
@@ -1904,3 +1914,126 @@ body.tribe-theme-twentysixteen .datepicker table {
...
+}
+
+.tribe-events-condensed-list .type-tribe_events {
+    border-bottom: 2px solid #eaeaea;
+}
+
+@media (max-width: 414px) {
+    .tribe-events-condensed-list .type-tribe_events {
+        padding-bottom: 1em;
+        margin-bottom: 1em;
```
Comment (borkweb) [inline src/resources/css/tribe-events-full.css]: I know the CSS files in the-events-calendar aren't a great reference for our future intentions, but going forward we're going to be alphabetizing CSS properties. We've started to do this in our other plugins and any new additions in here we'd love to see the properties alphabetized, too :grinning:

If you could make that change, that'd be great!

File: src/resources/css/tribe-events-full.css [hunk 2]
```diff
@@ -1904,3 +1914,126 @@ body.tribe-theme-twentysixteen .datepicker table {
...
+table.tribe-events-calendar-condensed-list {
+    border-collapse: collapse;
+    clear: both;
+    font-size: 80%;
+    table-layout: fixed;
+    width: 100%;
+    border: none;
+}
+
+@media (max-width: 414px) {
```
Comment (borkweb) [inline src/resources/css/tribe-events-full.css]: I would suggest wrapping all styles for `max-width: 414px` in the same media query.

File: src/resources/css/tribe-events-full.css (hunk 1 above)
Comment (trishasalas) [inline src/resources/css/tribe-events-full.css]: love this! done :)

File: src/resources/css/tribe-events-full.css (hunk 2 above)
Comment (trishasalas) [inline src/resources/css/tribe-events-full.css]: commenting as done for my own reference

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [content.php:43](https://github.com/moderntribe/the-events-calendar/blob/df373a3af12048e1d016dd0b312d1e76e458839b/src/views/list-condensed/content.php#L43)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [content.php:46](https://github.com/moderntribe/the-events-calendar/blob/df373a3af12048e1d016dd0b312d1e76e458839b/src/views/list-condensed/content.php#L46)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [content.php:49](https://github.com/moderntribe/the-events-calendar/blob/df373a3af12048e1d016dd0b312d1e76e458839b/src/views/list-condensed/content.php#L49)
:warning: | Expected next thing to be an escaping function (like esc_html_e() or esc_attr_e()), not '_e' | [content.php:52](https://github.com/moderntribe/the-events-calendar/blob/df373a3af12048e1d016dd0b312d1e76e458839b/src/views/list-condensed/content.php#L52)

> via jenkins/codesniffer-phpcs

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [content.php:43](https://github.com/moderntribe/the-events-calendar/blob/359e3681205743369cd9f83c799c3f4c649e3714/src/views/list-condensed/content.php#L43)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [content.php:46](https://github.com/moderntribe/the-events-calendar/blob/359e3681205743369cd9f83c799c3f4c649e3714/src/views/list-condensed/content.php#L46)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [content.php:49](https://github.com/moderntribe/the-events-calendar/blob/359e3681205743369cd9f83c799c3f4c649e3714/src/views/list-condensed/content.php#L49)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [content.php:52](https://github.com/moderntribe/the-events-calendar/blob/359e3681205743369cd9f83c799c3f4c649e3714/src/views/list-condensed/content.php#L52)

> via jenkins/codesniffer-phpcs

**Pull request closed without merging**